apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/style.scss

```diff
@@ -11,7 +11,15 @@
 	padding: 0;
 }
 
+.components-popover.block-editor-block-list__block-popover .components-popover__content .block-editor-block-contextual-toolbar[data-type='a8c/template'] {
```

Comment:
Fantastic!  I totally missed the data-type on there before when I was trying to hide this annoying thing. 🤣

Reply:
yeah, it was even tough to find that div in the first place!